style.css

```diff
@@ -15,11 +15,11 @@
   padding: 0;
 }
```

Comment:
I apologize if no reviewer has previously mentioned this, but your fonts don't match those of the template design. Your project should not have been approved without this change in the first place, so please make sure you import the necessary fonts to match those of the Figma design.

<img width="557" alt="Screen Shot 2022-07-07 at 21 48 24" src="https://user-images.githubusercontent.com/82426682/177907037-48f15bc0-3eb0-4c94-9f2f-2fda118aa2ab.png">

Reply:
Agreed wholeheartedly. Nevertheless, aberration has been normalized.